addons/escoria-core/game/core-scripts/esc/commands/change_scene.gd

```diff
@@ -93,16 +108,25 @@ func run(command_params: Array) -> int:
 	# Load room scene
 	var room_scene = res_room.instance()
 	if room_scene:
+		if command_params[1] \
+		and escoria.event_manager._running_event.name == "debug":
```

Comment:
Indent this two times as well.

Reply:
What's the "debug" event here?

Reply:
This is the event used by the room selector. Should we use another name maybe?

Reply:
Yes, "debug" is rather confusing. Why not directly go with "room_selector" or something?

Reply:
Let's go with that ok.